### PR TITLE
Update kafka to 1.1.1

### PR DESCRIPTION
**Pull request by jrpilat**
I think I did this right, tests passed locally.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -38,6 +38,8 @@ matrix:
     env: KAFKA_VERSION=1.0.1
   - scala: 2.11
     env: KAFKA_VERSION=1.1.0
+  - scala: 2.11
```
Comment (sscaling) [inline .travis.yml]: This should just replace `1.1.0`. `1.1.1` is the latest version of the `1.1.x branch`.

**Comment by sscaling** [comment]
Thanks!

File: Dockerfile [hunk 2]
```diff
@@ -1,6 +1,6 @@
 FROM openjdk:8u171-jre-alpine
 
-ARG kafka_version=1.1.0
+ARG kafka_version=1.1.1
```
Comment (jaceklaskowski) [inline Dockerfile]: If this Dockerfile uses Kafka 1.1.1 with Scala 2.12 (which is recommended) why does `.travis.yml` use 2.11? Just curious.

Comment (sscaling) [reply Dockerfile]: http://kafka.apache.org/downloads recommends 2.11 with version 1.1.1. Is 2.12 recommended use documented anywhere else? 

Travis ci always passes build arguments, so the default values should probably be removed as they are redundant.

Comment (jaceklaskowski) [reply Dockerfile]: Sorry for the "noise". I thought that 2.12 was the recommended version.

Comment (sscaling) [reply Dockerfile]: no problem. Good to have another pair of eyes. And it highlights the fact that we should probably remove the default values. Will address as part of https://github.com/wurstmeister/kafka-docker/issues/363

**Pull request merged**